wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisartDTO.java

```diff
@@ -0,0 +1,14 @@
+package de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.rest.ereignis;
+
+import de.muenchen.oss.wahllokalsystem.vorfaelleundvorkommnisseservice.service.EreignisartModel;
+
+public enum EreignisartDTO {
+    VORFALL, VORKOMMNIS;
+
+    public static EreignisartDTO fromModel(EreignisartModel model) {
+        return switch (model) {
+        case VORFALL -> VORFALL;
+        case VORKOMMNIS -> VORKOMMNIS;
+        };
+    }
```

Comment:
Bitte in den Mapper oder die Testdatenfactory verlagern.

Reply:
siehe commit [b1a086e](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/b1a086e3245ecd720cc3083f9e237c025716cf4d)